/**:
  ros__parameters:
    pose_message_type: "PoseStamped"
    pose_sub_topic: "pose"
    pose_array_pub_topic: "filtered_pose_array"
    target_frame: "odom"
